lib/shopify_app/scripttags_manager.rb

```diff
@@ -3,10 +3,12 @@ class ScripttagsManager
     class CreationFailed < StandardError; end
 
     def self.queue(shop_domain, shop_token, scripttags)
-      ShopifyApp::ScripttagsManagerJob.perform_later(
-        shop_domain: shop_domain,
-        shop_token: shop_token,
-        scripttags: scripttags
+      ShopifyApp::ScripttagsManagerJob
+        .set(queue: ShopifyApp.configuration.webhooks_manager_queue_name)
```

Comment:
I don't actually need dynamic config but I had an issue trying to implement it like described in https://github.com/Shopify/shopify_app/issues/297.
 I'm setting the `config.webhooks_manager_queue_name` value in the `initializers/shopify_app.rb`. The issue is that it is still `nil` so it uses the default value when the app loads `webhooks_manager_job.rb` file.